src/models/propulsion/FGTurbine.h

```diff
@@ -41,6 +41,9 @@ INCLUDES
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%*/
 
 #include "FGEngine.h"
+#include "math/FGFunction.h"
+#include "math/FGRealValue.h"
+
```

Comment:
You don't need to include these headers because the class `FGTSFC` will store the `<tsfc>` as a `double`
```suggestion
```